src/skip-to-content/skip-to-content.scss

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+
+
+@mixin iui-skip-to-content {
+  @include iui-blur;
+  @include iui-focus($offset: 2px, $thickness: 2px);
+
+  border-radius: 99999px;
+  color: t(iui-color-foreground-accessory);
+  padding: ($iui-baseline * 0.75) $iui-m;
+  text-decoration: none;
+  position: absolute;
+  text-align: center;
+  left: 50%;
+  transform: translateX(-50%) translateY(-170%);
```

Comment:
Can you explain this magic number (170)? Or did you just pick an arbitrary off-screen value?

Reply:
This is the nearest round number that just barely hides the component. The closer the component is to the edge of the screen, the faster it will appear when tabbed to.

Reply:
> The closer the component is to the edge of the screen, the faster it will appear when tabbed to.

I dont think that is necessarily true. The component will always appear in 200ms because that's the duration of the transition.

I'm just wondering if there's a better way to hide this link. Maybe using visually hidden styles 🤔

Have you done the research on common ways skip links are usually implemented?

Reply:
I just tested this and that is what happens. I see what you're saying, but the component will be partially visible to the user faster if it's closer to the edge, not that the component will be finished moving sooner.

I did do the research. [Here is the WebAIM article](https://webaim.org/techniques/skipnav/) Virginie posted on the issue. [Further information about invisible content is located here.](https://webaim.org/techniques/css/invisiblecontent/) WebAIM is using a similar setup for their own site.

I am not sure what you mean by "visually hidden styles" exactly. If you have a specific suggestion please share it.

Reply:
I don't see any CSS in the first link. It is just talking about skip links at a high-level. When I say "research" I was talking about practical CSS that is recommended. This can include how-to articles as well as actual code that is used in real-world websites or component libraries. Here is an example: https://github.com/reach/reach-ui/tree/main/packages/skip-nav

The second link is about invisible content in general, and it even includes the "visually hidden" technique (tldr: hide content using `clip-path` and place it in a 1x1 box). [https://webaim.org/techniques/css/invisiblecontent/#techniques](https://webaim.org/techniques/css/invisiblecontent/#:~:text=receive%20keyboard%20focus.-,css%20clip,-%7Bclip%3A%20rect)

Reply:
I talked to Jon when determining the design for this component - he likes that the link moves from off-screen to on-screen vertically.

WebAIM specifically recommends this style:
> Probably the most accessible method for visually hiding a skip link is to hide it off screen with CSS, then cause it to be positioned on screen when it receives keyboard focus.
> One possible drawback to this approach is that the sudden appearance of a link that was previously invisible will be unexpected and could potentially confuse the sighted keyboard user. The WebAIM.org site address this by implementing CSS transitions to animate the focused "skip" link from above the top of the page (as opposed to off-screen left) to the top left corner of the page, then back above the page when focus is lost. This ensures that the link is highly visible on screen for a longer period of time.

I looked into using clip or clip-path, and it seems that the animation style would be more pop-onto-the-page rather than slide-onto-the-page. This is what skip-nav does, and tabbing quickly can mean missing the link. I suppose it could have a fade-in and/or fade-out effect, but in either case it seems that it would be less visually noticeable than the component moving.